setup.py

```diff
@@ -53,10 +53,11 @@ def get_snakemake_pkg(extras=""):
     "click>=7.0",
     "fs>=2.0",
     "jsonschema[format]>=3.0.1,<4.0.0",
-    "kombu>=4.6,<4.7",
+    "kombu>=4.6",
     "mock>=3.0,<4",
     "PyYAML>=5.1,<6.0",
     "Werkzeug>=0.14.1",
+    "pygraphviz<1.8",
```

Comment:
Not sure if this is the best decision. Maybe upgrading a package that brings `pygraphviz` would be a better idea, but, I guess this comes from `yadage` so unlikely there are any options.